docs/getting-started/list-filtering/sqlalchemy.rst

```diff
@@ -0,0 +1,283 @@
+==================
+SQLAlchemy Adapter
+==================
+
+The ``sqlalchemy_oso`` library can enforce policies over SQLAlchemy models.
```

Comment:
Should it be "The ``sqlalchemy-oso`` library" since that's what you pip install it as? Library vs. module distinction